.gitignore

```diff
@@ -411,6 +411,7 @@ modules.xml
 # Project specific ignores
 /build.properties
 /data/
+**/respondr/defaultSkin.css
```

Comment:
Please -- let's not start doing that kind of thing.

EVERY product of the build goes into `build/`.